range_tombstone_list.cc

```diff
@@ -375,13 +376,13 @@ range_tombstone_list::reverter::insert(range_tombstones_type::iterator it, range
 
 range_tombstone_list::range_tombstones_type::iterator
 range_tombstone_list::reverter::erase(range_tombstones_type::iterator it) {
-    _ops.reserve(_ops.size() + 1);
+    amortized_reserve(_ops, _ops.size() + 1);
```

Comment:
Isn't this the same strategy `std::vector` follows by default? I think this code would not have been quadratic without the manual reserve call.

Reply:
Would just removing the reserve altogether also fix this?

Reply:
The reason we need to reserve() is so that push_back() does not throw.

Reply:
See @kbr-'s comment below.